README.md

```diff
@@ -168,14 +168,12 @@ final class MyController {
         let message: String
     }
     func sendUserEmailFromJSON(_ req: Request) throws -> EventLoopFuture<HTTPStatus> {
-        return try req.content.decode(EmailData.self)
-            .flatMap { (emailData)->EventLoopFuture<SES.SendEmailResponse> in
-                let destination = SES.Destination(toAddresses: [emailData.address])
-                let message = SES.Message(body:SES.Body(text:SES.Content(data:emailData.message)), subject:SES.Content(data:emailData.subject))
-                let sendEmailRequest = SES.SendEmailRequest(destination: destination, message: message, source:"awssdkswift@me.com")
+        let emailData = try req.content.decode(EmailData.self)
+        let destination = SES.Destination(toAddresses: [emailData.address])
+        let message = SES.Message(body:SES.Body(text:SES.Content(data:emailData.message)), subject:SES.Content(data:emailData.subject))
+        let sendEmailRequest = SES.SendEmailRequest(destination: destination, message: message, source:"awssdkswift@me.com")
 
-                return client.sendEmail(sendEmailRequest)
-            }
+        return client.sendEmail(sendEmailRequest)
```

Comment:
would it be better to use `req.client` here to avoid the hop?

Reply:
You are thinking about soto v5 here surely

Reply:
Ah ok I see. Yeah ok leave it as is. I guess the Vapor way would be to do `req.ses.sendEmail` but fitting that into the docs is probably out of scope

Reply:
Yeah the V5 docs cover this in a lot more detail, but for V4 I'll stick to this